phabfive/maniphest.py

```diff
@@ -81,10 +96,149 @@ def create_from_config(self, config_file, dry_run=False):
                     has_errors = True
 
             if has_errors:
-                log.error(f"Could not find one or more specified projects in phabricator instance. All available project names")
+                log.error(f"Could not find one or more specified users, projects in phabricator instance. All available project names")
                 log.error(list(project_name_to_id_mapping.keys()))
                 return (False, None)
 
+        ### Validate owner and all subscribers (=users) is correct data types and exists in Phabricator instance
+        #create a new construct to consolidate all users for owner and subscriber
+        users_query = {}
+        users_query["data"] = []
+
+        r = self.phab.user.search()
+        length = len(r.data)
+        afterCursor = r['cursor']['after']
+        users_query["data"].extend(r.data)
+
+        while (length >= 100 and afterCursor is not None):
+            r = self.phab.user.search(
+                after=afterCursor)
+            length = len(r.data)
+            afterCursor = r['cursor']['after']
+            users_query["data"].extend(r.data)
+
+        user_name_to_id_mapping = {
+            user["fields"]["username"]: user["phid"]
+            for user in users_query["data"]
+        }
+
+        log.debug("All Phabricator user names")
+        log.debug(user_name_to_id_mapping)
+
+        for index, ticket_data in enumerate(data["tickets"]):
+            if ("owner" not in ticket_data):
+                break
+
+            has_errors = False
+
+            if ticket_data["owner"] not in user_name_to_id_mapping:
+                log.error(f"Owner User name '{ticket_data['owner']}' not found in Phabricator instance")
+                has_errors = True
+
+            if has_errors:
+                log.error(f"Could not find one or more specified users, projects in phabricator instance. All available project names")
+                log.error(list(user_name_to_id_mapping.keys()))
+                log.error(f"All available users names")
+                log.error(list(user_name_to_id_mapping.keys()))
```

Comment:
Duplicate code here by creating same list twice, could be simplified

Reply:
nice catch - is a bug - line 140 is about the project list while 142 user list.